apps/src/netsim/NetSimEntity.js

```diff
@@ -38,13 +36,6 @@ var NetSimEntity = module.exports = function (shard, entityRow) {
    * @type {number}
    */
   this.entityID = entityRow.id;
-
-  /**
-   * Change event fired when entity's state changes in a way that
-   * should be reported.
-   * @type {ObservableEvent}
-   */
-  this.onChange = new ObservableEvent();
```

Comment:
This change removes the only consumer of `NetSimEntity.onChange` so I figured it was time to tear it out.